tests/ctst/features/resource-policies/regen.js

```diff
@@ -0,0 +1,168 @@
+/**
```

Comment:
Can we add more comments on this file, which can act as a guide to someone new contributing to this?

Reply:
Done: https://github.com/scality/Zenko/pull/2023/commits/a3d3fb61cd5004370bdcf10f0f2743158393f5a9